fold-and-r.csl

```diff
@@ -129,6 +129,15 @@
             </group>
           </group>
         </else-if>
+        <else-if type="speech">
+          <group delimiter=" ">
+            <text macro="title" prefix="'" suffix="'"/>
```

Comment:
did we ever talk about why you're using straight quotes in the affixes rather than quotes="true" (which would be more elegant and produce nice curly quotes?